### ISSUE TITLE
Final Peer Review - Yichen Shao ys2233 

### ISSUE THREAD

**Issue by yichenshao** [manuhanu/Predict-Bitcoin#9]
Their goal is to use several machine learning algorithms to predict the price or direction of Bitcoin. They used dataset from CryptoDataDownload.com which contains data from 2017 to 2021. They aim to make appropriate trading decisions on Bitcoin using machine learning models.

Things I like about the project:

I like how they explained a few features and terms about Bitcoin to the audience because this project is really domain specific.
They considered using Google Trends data as one of the indicators. It utilized the wisdom of crowd(crowd sourcing) techniques.
They computed features' correlation heat map to identify any highly correlated feature pairs. It's really crucial when using linear models.
They used several metrics to evaluate the performance of their models: precision, accuracy.
They backtest their model and introduced another metrics: SharpeRatio to determine if their models are trustworthy.

Things I think need to improve:

Standardizing or normalizing the data for linear model could be really crucial. Because each feature has a different scale, and it could make the linear model fail to capture useful information.
For each feature, there could be outliers in some degree, by handling outliers could possibly improve the performance.
They could form a more informative method to identify overfitting and validation like k-fold cross validation.
They could use grid search, random search, or bayesian optimization on hyper-parameters tuning. It could sometimes help improve the results a lot.
